Python/Product/Uwp.Interpreter/Uwp.Interpreter.csproj

```diff
@@ -43,6 +43,14 @@
       <Project>{A85D479D-67A9-4BDB-904A-7D86DAF68A6F}</Project>
       <Name>Analysis</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Common\Common.csproj">
```

Comment:
As I said above, I hope we can remove this reference. If there's something that you need from it, tell us and we'll look at making it public/stable API rather than internal API.